packages/eas-cli/src/prompts.ts

```diff
@@ -1,3 +1,7 @@
+// @ts-ignore
+// the docs mention we can create custom prompts by sublclassing built-in prompts:
+// https://github.com/enquirer/enquirer#-custom-prompts
+import MultiSelect from 'enquirer/lib/prompts/multiselect';
```

Comment:
```suggestion
import { MultiSelect } from 'enquirer';
```
Also, I don't see what this comment and link are there, initially I assumed that this is an explanation for using that direct import, but it does not seem necessary.

Reply:
good catch! the typing in the module is non-existant, so I got a ts error when I tried to import multiselect from the package in the way you suggest. However, when I actually runt he code, it does import properly. I'll remove the comment and adopt this import style.